src/main/resources/db/changelog/changes/create-books-categories.yaml

```
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+    id: createTable_Book_Category
+    author: Viacheslav Torbin
+    changes:
+      - createTable:
+          tableName: book_category
+          columns:
+            - column:
+                name: book_id
+                type: bigint
+                constraints:
+                  foreignKeyName: FK_BookCategory_Book
```

```suggestion
                  foreignKeyName: fk_book_category_book
```
use snake case  also you can do it like here:
```
- createTable:
            tableName: user_role
            columns:
              - column:
                  name: user_id
                  type: BIGINT
                  constraints:
                    nullable: false
                    references: users(id)
                    foreignKeyName: fk_user_role_user_id
                    deleteCascade: true
                    primaryKey: true
              - column:
                  name: role_id
                  type: BIGINT
                  constraints:
                    nullable: false
                    references: roles(id)
                    foreignKeyName: fk_user_role_role_id
                    deleteCascade: true
                    primaryKey: true
            primaryKey:
              columnNames: user_id, role_id
```